src/System.Windows.Forms/src/System/Windows/Forms/MonthCalendar.cs

```diff
@@ -1649,12 +1649,13 @@ protected override void OnHandleCreated(EventArgs e)
             {
                 firstDay = (int)firstDayOfWeek;
             }
-            SendMessage((int)ComCtl32.MCM.SETFIRSTDAYOFWEEK, 0, firstDay);
+
+            User32.SendMessageW(this, (User32.WM)ComCtl32.MCM.SETFIRSTDAYOFWEEK, (IntPtr)0, (IntPtr)firstDay);
```

Comment:
This looks odd. Should this be:
```suggestion
            User32.SendMessageW(this, (User32.WM)ComCtl32.MCM.SETFIRSTDAYOFWEEK, IntPtr.Zero, (IntPtr)firstDay);
```